.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+<!-- Thanks for taking the time to write this Pull Request ❤️ -->
+
+## 🚀 Description
+
+<!-- Describe your changes in detail -->
+
+## 📄 Motivation and Context
+
+<!-- Why is this change required? What problem does it solve? -->
+<!-- If it fixes an open issue, please link to the issue here. -->
+
+## 🧪 How Has This Been Tested?
```

Comment:
I would remove the paragraph. We have a CI server that runs the tests to make sure everything is tested.

That the tests need to be run, should be a friendly reminder, not necessarily something to describe out.

Reply:
You are right. I removed the testing part.